Minor Oracle Linux 8/9 corrections

Hi,

As posted here : https://discourse.maas.io/t/informative-post-about-ol8-and-ol9-image-creation-with-latest-packer-maas/7714
I think it would be better to perform the changes:

1- Change the ol8/ol9,ks.in name in the http folder to ol8.ks/ol9.ks respectively. Otherwise packer throws an error because it is searching for that name. Or at least state in the README that this step should be done

2- If the user doesn't need a proxy he won't set the ${KS_PROXY} value and the image fails to create. In my case i just removed that variable from the kickstart file but I guess it should be corrected in a more elegant way

3- Using the make command instead of packer build to create the image fails with this error:
```
_current_deps := libnbd-bin nbdkit packer fuse2fs
make: _current_deps: No such file or directory
make: *** [Makefile:15: check-deps] Error 127
```
I guess we are missing this function/file to perform the check

4- The README should be updated as such:
`packer init . (missing the dot)`

and set the name for instance to custom/ol-8.8 and custom /ol-9.2 respectively in the maas command below. Otherwise it won't work due to the error “{“name”: [“Unsupport operating system ol, supported operating systems: [‘ubuntu’, ‘ubuntu-core’, ‘bootloader’, ‘centos’, ‘rhel’, ‘custom’, ‘windows’, ‘suse’, ‘caringo’, ‘esxi’]”]}”:
```

maas $PROFILE boot-resources create \
    name='custom/ol-8.8' title='Oracle Linux 8.8' \
    architecture='amd64/generic' filetype='tgz' \
    content@=ol8.tar.gz
```


What do you think about these suggestions?

Thanks
BR

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ Read more about how [custom images](https://maas.io/docs/how-to-customise-images
 
 ## Existing templates
 
-| **OS**          | **Maturity Level** | **MAAS Version** | 
+| **OS**          | **Maturity Level** | **MAAS Version** |
 |-----------------|:------------------:|:-----------------|
 | CentOS 6        | EOL                | >= 1.6           |
 | CentOS 7        | Stable             | >= 2.3           |
@@ -20,8 +20,8 @@ Read more about how [custom images](https://maas.io/docs/how-to-customise-images
 | Debian 10       | Beta               | >= 3.3           |
 | Debian 11       | Beta               | >= 3.3           |
 | Debian 12       | Beta               | >= 3.3           |
-| OL8             | Beta               | >= 3.5           |
-| OL9             | Beta               | >= 3.5           |
+| OL8             | Alpha              | >= 3.5           |
+| OL9             | Alpha              | >= 3.5           |
 | RHEL 7          | EOL                | >= 2.3           |
 | RHEL 8          | Stable             | >= 2.7           |
 | RHEL 9          | Beta               | >= 3.3           |
@@ -37,6 +37,7 @@ Read more about how [custom images](https://maas.io/docs/how-to-customise-images
 ### Maturity level
 
 * Templates marked as *EOL* are OSes that are no longer supported by the upstream maintainer, and **are not recommended for new deployments**. These systems don't receive security updates and mirrors can become permanently offline at any moment.
+* *Alpha* templates require packages that are not yet generally available, e.g. an unreleased MAAS or Curtin version. These should not be used in production environments.
 * *Beta* templates should work but we still don't have enough successful deployment reports to regard it as *Stable*.
 
 ### Output

--- a/ol8/Makefile
+++ b/ol8/Makefile
@@ -11,8 +11,7 @@ export PACKER_LOG KS_PROXY
 
 all: ol8.tar.gz
 
-check-deps:
-	$(call check_packages_deps)
+$(eval $(call check_packages_deps))
 
 ol8.tar.gz: check-deps clean http/ol8.ks
 	${PACKER} init ol8.pkr.hcl && ${PACKER} build ol8.pkr.hcl

--- a/ol8/README.md
+++ b/ol8/README.md
@@ -46,7 +46,8 @@ be in packer-maas/ol8, where this file is located. Once in packer-maas/ol8
 you can generate an image with:
 
 ```shell
-packer init
+envsubst '$${KS_PROXY}' < http/ol8.ks.in | tee http/ol8.ks
+packer init .
 PACKER_LOG=1 packer build .
 ```
 

--- a/ol9/Makefile
+++ b/ol9/Makefile
@@ -11,8 +11,7 @@ export PACKER_LOG KS_PROXY
 
 all: ol9.tar.gz
 
-check-deps:
-	$(call check_packages_deps)
+$(eval $(call check_packages_deps))
 
 ol9.tar.gz: check-deps clean http/ol9.ks
 	${PACKER} init ol9.pkr.hcl && ${PACKER} build ol9.pkr.hcl

--- a/ol9/README.md
+++ b/ol9/README.md
@@ -46,7 +46,8 @@ be in packer-maas/ol9, where this file is located. Once in packer-maas/ol9
 you can generate an image with:
 
 ```shell
-packer init
+envsubst '$${KS_PROXY}' < http/ol9.ks.in | tee http/ol9.ks
+packer init .
 PACKER_LOG=1 packer build .
 ```
 

